app/lib/frontend/static_files.dart

```diff
@@ -333,3 +350,33 @@ Future<void> updateWebCssBuild() async {
     throw Exception(message);
   }
 }
+
+@visibleForTesting
```

Comment:
This is exported from this library and used outside testing, I see it used in `app/lib/frontend/handlers/misc.dart`

Reply:
Ah, removed.